src/groupeddataframe/splitapplycombine.jl

```diff
@@ -653,24 +653,9 @@ function _combine(gd::GroupedDataFrame,
     # a correct index is stored in idx variable
 
     @sync for i in eachindex(trans_res)
-        @spawn begin
-            col_idx = trans_res[i].col_idx
-            col = trans_res[i].col
-            if keeprows && col_idx !== idx_keeprows # we need to reorder the column
-                newcol = similar(col)
-                # we can probably make it more efficient, but I leave it as an optimization for the future
-                gd_idx = gd.idx
-                k = 0
-                # consider adding @inbounds later
-                for (s, e) in zip(gd.starts, gd.ends)
-                    for j in s:e
-                        k += 1
-                        newcol[gd_idx[j]] = col[k]
-                    end
-                end
-                @assert k == length(gd_idx)
-                trans_res[i] = TransformationResult(col_idx, newcol, trans_res[i].name, trans_res[i].optional)
-            end
+        let i=i
```

Comment:
How about using `$i` instead?

Reply:
I was not sure the Julia 1.0 version of the macro supports `$i`. And for sure Julia 1.1, 1.2 and 1.3 do not support it.